test/unit/app/test_models.py

```diff
@@ -159,6 +159,11 @@ def test_valid_tfa_token(authed_sempo_admin_user):
             is not None)
 
 
+def test_get_most_relevant_transfer_usage(authed_sempo_admin_user):
```

Comment:
This just checks if a list is returned Testing the complete functionality requires a lot of though. (Creating multiple user, transer accounts, credits transfers, rainbows more stuff?).

Is it worth it?

Reply:
what kind of manual testing did you do? i thiiink it's okay to not thoroughly test it since it's not super critical functionality

Reply:
I filled the database using seed_dev and checked if it returned what i expected -> So the top most used transfer filled up with the default

And i did a check if it just return the just the default when you have a new user.

Reply:
yeah i think the effort-to-reward ratio for testing this whole thing doesn't make sense just yet. At some point we should probably build a fixture that creates a lot of transfers for testing other aspects, and at that point testing the rest of this will be easy (though not really a unit test) - maybe just throw in a TODO to remind us that this test is lightweight?